controllers/NUgus_Controller/CMakeLists.txt

```
@@ -10,8 +10,5 @@ add_executable(nugus_controller ${controller_sources})
 # Link against the webots target (this will also add any necessary include directories to our target)
 target_link_libraries(nugus_controller PRIVATE webots::webots)
 
-# Copy the built executable to the project root directory
-add_custom_command(TARGET nugus_controller POST_BUILD COMMAND ${CMAKE_COMMAND} -E
-  copy ${CMAKE_CURRENT_BINARY_DIR}/nugus_controller ${CMAKE_SOURCE_DIR}
-)
-
+set_target_properties(nugus_controller PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+                                       OUTPUT_NAME nugus_controller)
```

Until we do the rename. Also, update the gitignore file 
```suggestion
set_target_properties(nugus_controller PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
                                       OUTPUT_NAME NUgus_Controller)
```